src/domains/my/detail/MyScrap.jsx

```
@@ -1,105 +1,131 @@
-  ]
+  const [scrapPosts, setScrapPosts] = useState([])
+  const [loading, setLoading] = useState(false)
+  const [sort, setSort] = useState('latest')
+  const navigate = useNavigate()
+  const { user } = useAuthStore()
+  const loggedInUserId = user ? user.userId : null
+
+  const formatDate = (dateStr) => (dateStr ? dateStr.slice(0, 10) : '')
+
+  useEffect(() => {
+    useEffect(() => {
```

ESLint

[error] React Hook "useEffect" cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)